x-pack/test/cloud_security_posture_functional/page_objects/findings_page.ts

```diff
@@ -52,50 +52,52 @@ export function FindingsPageProvider({ getService, getPageObjects }: FtrProvider
       testSubjects.click(type === 'failed' ? 'distribution_bar_failed' : 'distribution_bar_passed'),
   };
 
-  const table = {
-    getElement: () => testSubjects.find('findings_table'),
+  const latestFindingsTable = {
+    getElement: (selector: string) => testSubjects.find(selector),
 
-    getHeaders: async () => {
-      const element = await table.getElement();
+    getFindingsContainer: () => testSubjects.find('findings_container'),
+
+    getHeaders: async (selector: string) => {
+      const element = await latestFindingsTable.getElement(selector);
       return await element.findAllByCssSelector('thead tr :is(th,td)');
     },
 
     getColumnIndex: async (columnName: string) => {
-      const headers = await table.getHeaders();
+      const headers = await latestFindingsTable.getHeaders('findings_table');
       const texts = await Promise.all(headers.map((header) => header.getVisibleText()));
       const columnIndex = texts.findIndex((i) => i === columnName);
       expect(columnIndex).to.be.greaterThan(-1);
       return columnIndex + 1;
     },
 
     getColumnHeaderCell: async (columnName: string) => {
-      const headers = await table.getHeaders();
+      const headers = await latestFindingsTable.getHeaders('findings_table');
       const headerIndexes = await Promise.all(headers.map((header) => header.getVisibleText()));
       const columnIndex = headerIndexes.findIndex((i) => i === columnName);
       return headers[columnIndex];
     },
 
     getRowsCount: async () => {
-      const element = await table.getElement();
+      const element = await latestFindingsTable.getElement('findings_table');
       const rows = await element.findAllByCssSelector('tbody tr');
       return rows.length;
     },
 
     getFindingsCount: async (type: 'passed' | 'failed') => {
-      const element = await table.getElement();
+      const element = await latestFindingsTable.getElement('findings_table');
```

Comment:
using `latestFindingsTable` from `findingsByResourceTable` is misleading. it'd be better if all three table objects used a single object for their common table actions and extend it as needed. here is an [example](https://github.com/elastic/kibana/commit/5b92c918ee06aaa4d20d15530615797d45723aea)